Form/EventListener/SortableCollectionListener.php

```diff
@@ -70,26 +71,17 @@ private function getEventDataKeys(FormEvent $event)
         return array_keys($event->getData());
     }
 
-    /**
-     * @param \Symfony\Component\Form\FormEvent $event
-     * @return array|null
-     */
-    private function getRememberedItemOrder(FormEvent $event)
+    private function getRememberedItemOrder(FormEvent $event): ?array
     {
         $formId = $this->getEventFormId($event);
 
-        if (isset($this->itemOrder[$formId])) {
-            return $this->itemOrder[$formId];
-        } else {
-            return null;
-        }
+        return isset($this->itemOrder[$formId])
```

Comment:
why not `return $this->itemOrder[$formId] ?? null` ?

Reply:
I was not aware it silences nonexistent key notices.